ovos_tts_plugin_piper/__init__.py

```diff
@@ -58,60 +63,105 @@ class PiperTTSPlugin(TTS):
                   'ryan-low',
                   'ryan-medium',
                   'lessac'],
-        'es': ['carlfm-x-low', 'mls_10246-low', 'mls_9972-low'],
-        'fi': ['harri-low'],
-        'fr': ['gilles-low', 'mls_1840-low', 'siwis-low', 'siwis-medium'],
+        'es': ['ald-medium',
+               'carlfm-x-low',
+               'davefx-medium',
+               'mls_10246-low',
+               'mls_9972-low',
+               'sharvard-medium'],
+        'fi': ['harri-low', 'harri-medium'],
+        'fr': ['gilles-low',
+               'mls_1840-low',
+               'siwis-low',
+               'siwis-medium',
+               'upmc-medium'],
+        'hu': ['anna-medium', 'berta-medium', 'imre-medium'],
+        'is': ['bui-medium', 'salka-medium', 'steinn-medium', 'ugla-medium'],
         'it': ['riccardo_fasol-x-low'],
+        'ka': ['natia-medium'],
         'kk': ['iseke-x-low', 'issai-high', 'raya-x-low'],
+        'lb': ['marylux-medium'],
         'ne': ['google-medium', 'google-x-low'],
         'nl': ['mls_5809-low',
                'mls_7432-low',
                'nathalie-x-low',
                'rdh-medium',
                'rdh-x-low'],
         'no': ['talesyntese-medium'],
-        'pl': ['mls_6892-low'],
-        'pt-br': ['edresson-low'],
+        'pl': ['darkman-medium',
+               'gosia-medium',
+               'mc_speech-medium',
+               'mls_6892-low'],
+        'pt': ['edresson-low', 'faber-medium', 'tugao-medium'],
```

Comment:
let's please separate pt-pt (tugao) and pt-br

phonetics are quite different and pt-pt voices are harder to find :)

Reply:
Done, but I believe it should be possible to choose any language variant. Maybe this check should be on the code level instead

Reply:
the code first will look for the full lang code with 4 letters, and then for the 2 letter variant

in ovos the lang is defined by the 4 letter code, so this ensures the best available model is used

also note this is only done if the user does not specify the model directly, it is just ensuring a valid default voice for each language, it is handling the case where a user enables the plugin without configuring it first